toolchain/check/operator.h

```diff
@@ -13,19 +13,29 @@ namespace Carbon::Check {
 
 struct Operator {
   llvm::StringLiteral interface_name;
+  llvm::ArrayRef<SemIR::InstId> interface_args = {};
   llvm::StringLiteral op_name = "Op";
 };
 
 // Checks and builds SemIR for a unary operator expression. For example,
-// `$operand` or `operand$`.
-auto BuildUnaryOperator(Context& context, Parse::AnyExprId node_id, Operator op,
-                        SemIR::InstId operand_id) -> SemIR::InstId;
+// `$operand` or `operand$`. If specified, `missing_impl_diagnoser` is used to
```

Comment:
I'm not quite sure what you're trying to get at here, I'm used to seeing `$operand` for shell expansions, and I'm not aware of it being valid in Carbon. [I think it's been used in the past as a placeholder, probably that's why `$`, but I'm not sure that's going to make sense to newcomers.] Maybe we can use `*`?
```suggestion
// `*operand` or `operand*`. If specified, `missing_impl_diagnoser` is used to
```

Reply:
FWIW, in C++ land, specifically WG21, `$` is commonly used as a placeholder symbol when describing unary- and binary-operator syntaxes that apply to multiple different operator symbols.

Not saying it's a great idiom, but that's what I suspect is going on here. Not sure what the best syntax is though.

Reply:
Yeah, this is `$` as in "some arbitrary operator". But in any case this is pre-existing. I'd be fine if the example were changed to use `*` instead.